packages/addons/service/rsyslog/source/bin/rsyslog.init

```diff
@@ -20,9 +20,95 @@
 . /etc/profile
 oe_setup_addon service.rsyslog
 
-mkdir -p "$ADDON_HOME"
+mkdir -p "$ADDON_HOME/conf"
 
-if [ ! -f "$ADDON_HOME/rsyslog.conf" ]
+ADDON_SETTINGS="$ADDON_HOME/settings.xml"
+ADDON_DEST="$ADDON_HOME/conf"
+RSYSLOG_IP_NAME_FQDN=`grep RSYSLOG_IP_NAME_FQDN $ADDON_SETTINGS | awk '{print $3 }' | sed -e "s,value=,," -e "s,\",,g"`
+RSYSLOG_PORT=`grep RSYSLOG_PORT $ADDON_SETTINGS | awk '{print $3 }' | sed -e "s,value=,," -e "s,\",,g"`
+RSYSLOG_PROTOCOL=`grep RSYSLOG_PROTOCOL $ADDON_SETTINGS | awk '{print $3 }' | sed -e "s,value=,," -e "s,\",,g"`
+RSYSLOG_KODI_LOGGING=`grep RSYSLOG_KODI_LOGGING $ADDON_SETTINGS | awk '{print $3 }' | sed -e "s,value=,," -e "s,\",,g"`
+RSYSLOG_JOURNAL_LOGGING=`grep RSYSLOG_JOURNAL_LOGGING $ADDON_SETTINGS | awk '{print $3 }' | sed -e "s,value=,," -e "s,\",,g"`
+
+if [ ! -f "$ADDON_DEST/kodi.conf" ]
+then
+  cp "$ADDON_DIR/config/kodi.conf" "$ADDON_DEST"
+fi
+
+cat > "$ADDON_HOME/rsyslog.conf" << 'RSYSLOG_CONF_BLOCK_PRE'
+# rsyslog v5 configuration file
```

Comment:
Not sure if this is still supported
http://www.rsyslog.com/doc/v5-stable/configuration/global/options/rsconf1_moddir.html